proxy-robots-module/src/main/java/orwell/proxy/zmq/ZmqMessageBroker.java

```diff
@@ -155,6 +155,7 @@ public void run() {
                 }
                 try {
                     // This is performed to avoid high CPU consumption
+                    //noinspection BusyWait
```

Comment:
Added in intelliJ to suppress a warning. Bad idea to keep it in the commited code ; will delete it